cmd/virt-apiserver-pki/README.md

```diff
@@ -0,0 +1,177 @@
+# API server PKI tool
+
+This tool is intended to simplify generation of the digital certificates
+needed by API services. The tool takes as parameters the API group and
+version of the API service and the the name of the Kubernetes secret
+where the files containing the generated private key and certificate
+should be stored. For example, to use the tool to create a certificate
+for the API service for group `myapi.io` and version `v1alpha1` the
+command line should look like this:
+
+```shell
+virt-apiserver-keytool \
+-logtosderr \
+-v 0 \
+-api-group=myapi.io \
+-api-version=v1alpha1 \
+-secret-name=my-apiserver-pki
+```
+
+If the secret doesn't contain the `apiserver.key` and `apiserver.crt`
+keys, or if they are empty, the tool generates a certificate signing
+request and sends it to Kubernetes for signing. Then it waits till the
+request is either approved or denied. If the request is approved the
+tool saves the private key and the generated certificate to the secret.
+If the request is denied then the tool finishes with an error code.
+
+## How to use it
+
+The tool needs a secret, but it doesn't it, it must be created manually:
```

Comment:
This doesn't quite flow...

Reply:
Should say "The tool needs a secret, but it doesn't *create* it, it ...", I will fix it.